test/functional/desired.js

```diff
@@ -14,6 +14,28 @@ function checkFeatureInEnv (envArg) {
   return !!feature;
 }
 
+function deepFreeze (object) {
```

Comment:
the latest version of appium support now has this method so it can be imported instead